packages/react-theme-provider/src/useThemeProviderClasses.tsx

```diff
@@ -13,7 +20,15 @@ export const useThemeProviderClasses = makeClasses(theme => {
         fontFamily: 'var(--body-fontFamily)',
         fontWeight: 'var(--body-fontWeight)',
         fontSize: 'var(--body-fontSize)',
+        MozOsxFontSmoothing: 'var(--body-mozOsxFontSmoothing)',
+        WebkitFontSmoothing: 'var(--body-webkitFontSmoothing)',
+      },
+      {
+        '& input': inheritFont,
```

Comment:
in [Fabric.styles](https://github.com/microsoft/fluentui/blob/master/packages/office-ui-fabric-react/src/components/Fabric/Fabric.styles.ts#L26), we also have `& button: inheritFont`. I don't think that's correct because our button defines the font styles and `button` is the root element. if setting `inherit`, the button styling will be incorrectly overwritten by `Fabric`. see [this pen](https://codepen.io/xugao/pen/OJNweZV)